src/Console/DirectiveCommand.php

```diff
@@ -59,28 +59,28 @@ class DirectiveCommand extends LighthouseGeneratorCommand
     /**
      * The required imports.
      *
-     * @var \Illuminate\Support\Collection<string>
+     * @var \Illuminate\Support\Collection<int, string>
      */
     protected $imports;
 
     /**
      * The implemented interfaces.
      *
-     * @var \Illuminate\Support\Collection<class-string>
+     * @var \Illuminate\Support\Collection<int, string>
```

Comment:
Shouldn't `class-string`s be FQCN? The _short names_ which are pushed in the `implements` property are just strings, because the can't be resolved as actual classes anymore.

Reply:
You are right, nevermind. Thanks for fixing this.